code_samples/data_migration/src/Migrations/Matcher/SectionIdentifierNormalizer.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace App\Migrations\Matcher;
+
+use eZ\Publish\API\Repository\Values\Content\Query\Criterion;
+use Ibexa\Platform\Bundle\Migration\Serializer\Normalizer\Criterion\AbstractCriterionNormalizer;
+use Webmozart\Assert\Assert;
+
+class SectionIdentifierNormalizer extends AbstractCriterionNormalizer
+{
+    public function __construct()
+    {
+        parent::__construct('section_identifier');
+    }
+
+    /**
+     * @param array<mixed> $data
+     * @param array<mixed> $context
+     */
+    protected function createCriterion(array $data, string $type, ?string $format, array $context): Criterion
```

Comment:
Once https://github.com/ibexa/migrations/pull/279 is merged this will become:

```suggestion
    protected function createCriterion(array $data, string $type, ?string $format, array $context): FilteringCriterion
```

Note: this also applies to `use` statement at the beginning of the file - `FilteringCriterion` is in an entirely different namespace.